README.md

```diff
@@ -2,11 +2,13 @@
 
 # Morpho Checkpointing Node
 
-This is the codebase of the checkpointing nodes for [Mantis](https://mantisclient.io/).
+Morpho is an [ECIP1097](https://ecips.ethereumclassic.org/ECIPs/ecip-1097)-based prototype implementation for securing ETC against 51% attacks by having a majority of a small set of trusted parties agree to insert checkpoint blocks at specific positions.
 
-For building and development, a recent Nix version supporting Flakes is recommended, see [here](https://nixos.wiki/wiki/Flakes#Installing_flakes) for installation instructions. The readme assumes such a newer version is available, however older Nix versions can still be used with slightly adjusted commands. Not using Nix is also a possibility but not explicitly shown.
+[User Documentation](./docs)
+
+This readme is only meant for developers.
 
-For an overview of how morpho is implemented, see [overview.md](./overview.md).
+For building and development, a recent Nix version supporting Flakes is recommended, see [here](https://nixos.wiki/wiki/Flakes#Installing_flakes) for installation instructions. The readme assumes such a newer version is available, however older Nix versions can still be used with slightly adjusted commands. Not using Nix is also a possibility but not explicitly shown.
```

Comment:
[here] -> [installation instructions here]
 for installation instructions. -> .